### PR TITLE
Add log for debugging unit test

### PR DESCRIPTION
**Pull request by jja725**
### What changes are proposed in this pull request?

Please outline the changes and how this PR fixes the issue.

### Why are the changes needed?

Please clarify why the changes are needed. For instance,
  1. If you propose a new API, clarify the use case for a new API.
  2. If you fix a bug, describe the bug.

### Does this PR introduce any user facing changes?

Please list the user-facing changes introduced by your change, including
  1. change in user-facing APIs
  2. addition or removal of property keys
  3. webui

### CODE REVIEW

**Comment by yuzhu** [comment]
you will need to follow https://howtodoinjava.com/log4j2/configure-log4j2-for-junit/ to see the logs

File: tests/src/test/java/alluxio/job/master/JobMasterIntegrationTest.java [hunk 1]
```diff
@@ -42,14 +42,17 @@
...
+import org.slf4j.LoggerFactory;
 
 import java.util.List;
 import java.util.concurrent.atomic.AtomicReference;
 
 /**
  * Integration tests for the job master.
  */
-public final class JobMasterIntegrationTest extends BaseIntegrationTest {
+public final class  JobMasterIntegrationTest extends BaseIntegrationTest {
```
Comment (apc999) [inline tests/src/test/java/alluxio/job/master/JobMasterIntegrationTest.java:54]:
```suggestion
public final class JobMasterIntegrationTest extends BaseIntegrationTest {
```

File: tests/src/test/java/alluxio/job/master/JobMasterIntegrationTest.java [hunk 2]
```diff
@@ -87,7 +90,7 @@ public void multipleTasksPerWorker() throws Exception {
 
     JobInfo jobStatus = mJobMaster.getStatus(jobId);
     assertEquals(2, jobStatus.getChildren().size());
-
+    LOG.info("Check job status for multipleTasks{}",jobStatus.getStatus());
```
Comment (apc999) [inline tests/src/test/java/alluxio/job/master/JobMasterIntegrationTest.java:93]:
```suggestion
    LOG.info("Check job status for multipleTasks{}", jobStatus.getStatus());
```

File: tests/src/test/java/alluxio/job/master/JobMasterIntegrationTest.java [hunk 3]
```diff
@@ -194,6 +197,8 @@ public void throttleJobWorkerTasks() throws Exception {
     mJobMaster.setTaskPoolSize(1);
 
     long jobId0 = mJobMaster.run(new SleepJobConfig(1));
+    JobInfo jobStatus = mJobMaster.getStatus(jobId0);
+    LOG.info("Check job status for throttleJobWorkerTasks{}",jobStatus.getStatus());
```
Comment (apc999) [inline tests/src/test/java/alluxio/job/master/JobMasterIntegrationTest.java:201]:
```suggestion
    LOG.info("Check job status for throttleJobWorkerTasks{}", jobStatus.getStatus());
```

File: tests/src/test/java/alluxio/job/master/JobMasterIntegrationTest.java [hunk 4]
```diff
@@ -233,7 +238,7 @@ public void cancel() throws Exception {
 
     JobInfo status = mJobMaster.getStatus(jobId);
     List<JobInfo> children = status.getChildren();
-
+    LOG.info("Check job status:{}", status);
```
Comment (apc999) [inline tests/src/test/java/alluxio/job/master/JobMasterIntegrationTest.java:241]:
```suggestion
    LOG.info("Check job status: {}", status);
```

**Review by apc999 — COMMENTED** [review]
@yuzhu  please check if these logging shall be in INFO level
@jja725 https://docs.alluxio.io/os/user/stable/en/contributor/Code-Conventions.html#logging-conventions fyi

**Pull request closed without merging**